Library/Formula/pmd.rb

```diff
@@ -2,23 +2,28 @@
 
 class Pmd < Formula
   homepage "http://pmd.sourceforge.net/"
-  url "https://downloads.sourceforge.net/project/pmd/pmd/5.1.3/pmd-src-5.1.3.zip"
-  sha1 "da6023d73e078444a7b76e8a7a0cfbeb421221e4"
+  url "https://downloads.sourceforge.net/project/pmd/pmd/5.3.2/pmd-src-5.3.2.zip"
+  sha1 "036cdb7583de3ea0d24e7486bf7b9f9f14bfb1ec"
+
+  depends_on "maven" => :build
+
+  # The mvn package target produces a .zip with all the jars needed for PMD
+  # This patch makes it produce a folder instead so installing the jars is straightforward
+  patch :DATA
```

Comment:
I looked into that, and opted against it since there was no mention of that being possible. I tried using a resource for it, but internally it uses downloaders that assume remote paths. I believe this is the cleanest, simplest solution.